### DISCUSSION TITLE
Add annotation api call to get an individual annotation by ID

### DISCUSSION THREAD

**Discussion by scottbock** [grafana/grafana#47722] [General]
We can PUT,PATCH,and DELETE by ID but not GET. 

We can only retrieve annotations by searching.

Our application would like to be able to GET by ID so that we can check some existing values before allowing PUT, PATCH or DELETE.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
